modules/jupyter-widget/src/utils.js

```diff
@@ -55,6 +63,8 @@ export function initDeck({mapboxApiKey, container, jsonInput, tooltip, onComplet
           getTooltip,
           container
         });
+        const warn = deck.log.warn;
+        deck.log.warn = wrapWarn(warn, handleWarning);
```

Comment:
Wrap `console.warn` instead of `deck.log.warn`. `log.warn` also dedupes messages.

Reply:
No support for piping console error messages?